pipelines/epinow2/generate_config.py

```diff
@@ -1 +1,46 @@
-pass
+import os
+from datetime import datetime
+
+from utils.epinow2.functions import (
+    generate_job_id,
+    generate_task_configs,
+    validate_args,
+)
+
+if __name__ == "__main__":
+    """
+    A script to generate `epinow2` configuration objects. This
+    will be invoked either by 1) a scheduled GH action using
+    default values, or 2) a user-triggered workflow with supplied
+    state, pathogen, report_date, and reference_date parameters.
+    This script is the entrypoint to the workflow that generates
+    configuration objects, validates them against a schema, and
+    writes them to Blob Storage.
+    """
+
+    # Pull run parameters from environment
+    state = os.environ.get("state", "all")
+    pathogen = os.environ.get("pathogen", "all")
+    report_date = os.environ.get(
+        "report_date", datetime.today().strftime("%Y-%m-%d")
```

Comment:
```suggestion
        "report_date", datetime.today().isoformat()
```
Not totally necessary, but I like using `.isoformat()`

Reply:
That said, actually, why not leave as `date` here? And why not parse reference_dates as `list[date]` as well? I know they get parsed into dates in `validate_args()`, but if there's an issue with the date format, it might be nice for the user to get more immediate feedback that it's bad, i.e. "fail fast"

It also makes it easier for us the devs to know that variable with `date` in the name are `date` types, not string types.

Reply:
Sounds good, yeah agreed, it'll be easier to just use dates throughout. Will update